rakelib/git_helpers.rake

```diff
@@ -11,7 +11,7 @@ module GitHelper
     elsif branch_exists
       Rake.sh('git', 'checkout', release_branch)
     else # create it
-      abort('Aborted, as not run from develop nor release branch') unless current_branch == 'develop' || Console.confirm("You are not on 'develop', nor already on '#{release_branch}'. Do you really want to cut the release branch from #{current_branch}?")
+      abort('Aborted, as not run from trunk nor release branch') unless current_branch == 'trunk' || Console.confirm("You are not on 'trunk', nor already on '#{release_branch}'. Do you really want to cut the release branch from #{current_branch}?")
```

Comment:
Note that this is related to the branch of the `release-toolkit` **itself**, as this script is used when doing the releases of the gem to RubyGems.
So this assumes that the `release-toolkit` repo itself will migrate to `trunk` between when this PR is merged and when we do the next version of it.

Which I think is the plan anyway (including given the changes you also made to the `Rakefile` which are similarly related) so I'm considering this was intentional… but given that this change is a bit different than the other changes in this PR (which are otherwise related to the branch of the _client repos_ the toolkit is used in), I just wanted to double-check this was the intent and highlight it 🙂

Reply:
Yes, it was completely intentional. I'll not merge this PR right now as I want to do the `trunk` rename as I am merging this probably on Wednesday.

Reply:
And thanks again for the review @AliSoftware 🙇